package.json

```diff
@@ -1,19 +1,21 @@
 {
   "name": "jest-vue-preprocessor",
-  "version": "0.2.0",
+  "version": "0.2.0-development",
   "description": "Preprocessor that allows importing of .vue files in jest tests",
   "engines": {
     "node": ">= 6.3"
   },
   "main": "index.js",
   "scripts": {
+    "commit": "git-cz",
     "lint": "eslint .",
     "test": "jest --no-cache",
-    "test:ci": "jest --coverage"
+    "test:ci": "jest --coverage",
```

Comment:
just a nitpick, but it's good practice to reuse script tasks. In this case:

```json
{
  "scripts":{ 
      "test": "jest",
      "test:watch": "npm t -- --watch",
      "test:coverage": "npm t -- --coverage"
   }
}
```

you may ask why?
well for instance take into consideration that your jest config is in separate file and you have to do `jest --config config/my-conf.json`

in this case you whould have to repeat that in every script

```json
{
  "scripts":{ 
      "test": "jest --config my/config.json",
      "test:watch": "jest --config my/config.json --watch",
      "test:coverage": "jest --config my/config.json --coverage"
   }
}
```

which is nod DRY at all